loadLayout.js

```diff
@@ -91,23 +91,42 @@ function newCharacter() {
     // race
     var raceText = document.createElement('a')
     raceText.innerHTML = 'Race: '
-    var race = document.createElement('input')
-    race.type = 'text-box'
-    race.placeholder = 'Lets make this a drop down'
-    race.id = 'playerRace'
+    var raceDropDown = document.createElement('select')
+    raceDropDown.id='raceType'
```

Comment:
Remember we will have to update the poUpSumbit js file to pull the values from your dropdowns now.